stdlib/buffer.gr

```diff
@@ -0,0 +1,742 @@
+/**
+ * @module Buffer: Utilities for working with buffers.
+ *
+ * @example import Buffer from "buffer"
+ *
+ * @since v0.4.0
+ */
+
+import Memory from "runtime/unsafe/memory"
+import WasmI32 from "runtime/unsafe/wasmi32"
+import Conv from "runtime/unsafe/conv"
+import Exception from "runtime/exception"
+import Int32 from "int32"
+import Bytes from "bytes"
+import String from "string"
+import { coerceNumberToWasmI32 } from "runtime/numbers"
+
+/**
+ * Buffers are an automatically growing data structure useful for storing and operating on a unknown number of bytes.
+ */
+record Buffer {
+  mut len: Number,
+  initialSize: Number,
+  mut data: Bytes,
+}
+
+let _SIZE_OFFSET = 4n;
+
+let _VALUE_OFFSET = 8n;
+
+let _8BIT_LEN = 1
+
+let _16BIT_LEN = 2
+
+let _32BIT_LEN = 4
+
+let _64BIT_LEN = 8
+
+/** Gets the size of a Bytes via its ptr */
```

Comment:
```suggestion
/* Gets the size of a Bytes via its ptr */
```
Shouldn't (I hope) be an issue for graindoc, but `/**` is actually the indicator to parse as a docblock instead of a multiline comment. Switch to `/*` to parse as the multiline comment.